lib/executor/ramping_arrival_rate.go

```diff
@@ -96,6 +96,7 @@ func (varc *RampingArrivalRateConfig) Validate() []error {
 	}
 
 	errors = append(errors, validateStages(varc.Stages)...)
+	errors = append(errors, validateVUShifts(varc.StartRate.Int64, varc.Stages)...)
```

Comment:
I've noticed that we call `validateVUShifts` here, but in practice, the `Stages` we receive here, in the context of the ramping arrival rate are "iterations per `timeUnit`" instead of VUs.

Thus, I would suggest  we consider a more fitting name for that function? Maybe `validateTargetShifts` considering in both cases (ramping vus, and ramping arrival rate, the parameter the users rely on is called "target").

What do you folks think?